include/stdbool.h

```diff
@@ -3,12 +3,17 @@
 
 #ifndef __cplusplus
 
-/**
-* Only define these for C, since C++ has its own bool support
-*/
+/*
+ * Only define these for C, since C++ has its own bool support
```

Comment:
Slight modification here: I would correct this to "Only defined for C, since C++ has its own bool support".

Reply:
Ok thanks. I will make those changes.